tests/integration/test_index_query/test_integration.py

```diff
@@ -53,9 +50,9 @@ def test_flow():
 
     print(f"Successfully started the flow: {FLOW_ID}. Let's send some query")
 
-    TEXT_MATCHED = get_results(query="anything will match the same")["search"]["docs"][
-        0
-    ]["matches"][0]["text"]
+    TEXT_MATCHED = get_results(query="text:anything will match the same")["search"][
+        "docs"
+    ][0]["matches"][0]["text"]
 
     print(f"document matched has the text: {TEXT_INDEXED}")
```

Comment:
Why are we having `TEXT_INDEXED` here instead of `TEXT_MATCHED`?

Reply:
an error I guess